select_examples_to_execute.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-#Collect all python examples to be executed
+# Collect all python examples to be executed
 echo "Running examples"
-set -ev
+# set -ev
```

Comment:
This is to make the script fail "better", but since it is giving problems it can go.